create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+# Gitops stack using DOKS and Flux CD
 
+This blueprint will guide you step by step on how to spin up a single DOKS (DigitalOcean Kubernetes) cluster and Flux CD for managing application deployments in a GitOps fashion. In the end we will also tell [Flux CD](https://fluxcd.io) to perform a basic deployment of the now ubiquitous `busybox` Docker application.
+
+[Terraform](https://www.terraform.io) was chosen to write `infrastructure as code` using declarative configuration files which allows for concise descriptions of resources using blocks, arguments, and expressions. In our guide it will be responsible with spinning up the DOKS (DigitalOcean Kubernetes) cluster as well as [Flux CD](https://fluxcd.io).
+
+[Flux CD](https://fluxcd.io) is used for managing the `Continuous Delivery` of applications inside the DigitalOcean Kubernetes cluster as well as a `Gitops` enabler.
+
+
+## Bootstrapping DOKS and Flux CD
+
+This section contains information about how we can bootstrap **DOKS** and **Flux CD** via **Terraform** using **Github** as a SCM provider and source of truth. The bootstrap process will create for us a DOKS cluster alongside with a new Github repository and branch where all the manifests used to provision [Flux CD](https://fluxcd.io) will be stored. The same repository will be used to store our Kubernetes custom application deployment manifests that will be managed automatically via [Flux CD](https://fluxcd.io).
+
+### Requirements
+
+1. A DigitalOcean [account](https://cloud.digitalocean.com) to create the access keys and provision the DOKS cluster.
+2. A [Github](https://github.com/) account.
+3. A git client has to be installed as well depending on the distro.
+  
+    E.g.: on MacOS it can be installed via Homebrew:
+    ```bash
+    brew info git
+    brew install git
+    ```
+
+4. [Terraform](https://www.terraform.io/downloads.html) must be installed next. 
+    
+    E.g.: on MacOS it can be installed via Homebrew:
+    ```bash
+    brew info terraform
+    brew install terraform
+    ```
+
+### Installation steps
+
+1. Start by creating a [Github personal access token](https://github.com/settings/tokens) that has the `repo` permissions set. The Terraform module provided by this blueprint will automatically create for us a Github repository and branch where all the required Kubernetes manifests will be stored. In order to achieve this we must create a few special environment variables so make sure to replace the `<>` placeholders accordingly:
+
+    ```bash
+    export TF_VAR_github_owner="<github_username>"                              # should be pretty self explanatory
```

Comment:
```suggestion
    export TF_VAR_github_owner="<github_username>"                              # GitHub username
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.